provision/acc_provision/apic_provision.py

```diff
@@ -2830,6 +3027,217 @@ def kube_tn(self, flavor):
+                                                                    )
+                                                                ]
+                                                            ),
+                                                        ),
+                                                        (
+                                                            "children",
+                                                            [
+                                                                collections.OrderedDict(
+                                                                    [
+                                                                        (
+                                                                            "vzEntry",
+                                                                            collections.OrderedDict(
+                                                                                [
+                                                                                    (
+                                                                                        "attributes",
+                                                                                        collections.OrderedDict(
+                                                                                            [
+                                                                                                (
+                                                                                                    "name",
+                                                                                                    "istio-9080",
+                                                                                                ),
+                                                                                                (
+                                                                                                    "etherT",
+                                                                                                    "ip",
+                                                                                                ),
+                                                                                                (
+                                                                                                    "prot",
+                                                                                                    "tcp",
+                                                                                                ),
+                                                                                                (
+                                                                                                    "sFromPort",
+                                                                                                    "9080",
```

Comment:
Could be a good idea to drive these values from input file eventually.

Reply:
Sure, as discussed - this may not be trivial; will sure give it a try in a later commit.